lib/pages/portal/service/website_provider.dart

```diff
@@ -206,7 +193,12 @@ class WebsiteProvider with ChangeNotifier {
             .map((doc) => WebsiteExtension.fromSnap(doc, ownerUid: uid)));
       }
 
-      initializeNumberOfVisits(websites);
+      final bool initializeReturn =
+          await _initializeNumberOfVisits(websites, uid);
+      if (!initializeReturn) {
+        const String failedReadMessage = 'Could not read favourite websites.';
```

Comment:
*Localization* means to add a string to the l10n folder in both Romanian and English. You seriously need to read the https://github.com/acs-upb-mobile/acs-upb-mobile/blob/master/CONTRIBUTING.md file.